config/routes.rb

```diff
@@ -11,7 +11,9 @@
 
   scope '/coordinator' do
     get '/deliver', to: 'coordinator#deliver', as: :coordinator_deliver
+    post '/deliver', to: 'coordinator#confirm', as: :coordinator_confirm
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.